reg_mainpage.php

```diff
@@ -28,7 +28,8 @@
         'mother'       => 1,
         'availableLangs' => [
             'en',
-            'pl'
+            'pl',
+            'fr'
```

Comment:
This should be removed until the entire translation is finished, only to prevent accidental usage by other people.